scripts/enable_automatic_updates.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+apt-get install unattended-upgrades curl
```

Comment:
add `apt-get update` first, then add `-y` argument to install

Reply:
Ok!